README.md

```diff
@@ -1,5 +1,7 @@
+\*Allotrope® is a registered trademark of the Allotrope Foundation; no affilication with the Allotrope Foundation is claimed or implied.
+
 # Introduction
-Welcome to `allotropy` -- a Python library for converting instrument data into Allotrope Simple Model (ASM).
+Welcome to `allotropy` -- a Python library by Benchling for converting instrument data into Allotrope Simple Model (ASM).
```

Comment:
should maybe be `into the Allotrope Simple Model` ? (add `the`)

Reply:
Done